### PR TITLE
minor fixes to R examples

### PR DESCRIPTION
**Pull request by timelyportfolio**
- add `x` in trace in reference and index, since no default is specified
- add `p` in `dplyr::filter` since not in a `%>%`

### CODE REVIEW

File: _posts/2015-07-30-r-index.md [hunk 1]
```diff
@@ -76,7 +76,8 @@ This allows us to mix data manipulation and visualization verbs in a [pure(ly) f
 
 ```r
 p %>%
-  add_trace(y = fitted(loess(uempmed ~ as.numeric(date)))) %>%
+  add_trace( x = date,
+             y = fitted(loess(uempmed ~ as.numeric(date)))) %>%
```
Comment (cldougl) [inline _posts/2015-07-30-r-index.md]: ahh @timelyportfolio this is so nit-picky (sorry!) could you bump that `x = ` up to the parenthesis `add_trace(x = ...` so it matches the format in the rest of this example

File: _posts/2015-07-30-r-index.md [hunk 2]
```diff
@@ -76,8 +76,8 @@ This allows us to mix data manipulation and visualization verbs in a [pure(ly) f
 
 ```r
 p %>%
-  add_trace( x = date,
-             y = fitted(loess(uempmed ~ as.numeric(date)))) %>%
+  add_trace(x = date,
+            y = fitted(loess(uempmed ~ as.numeric(date)))) %>%
```
Comment (cldougl) [inline _posts/2015-07-30-r-index.md]: 👍

**Comment by cldougl** [comment]
all good! 💃

File: _posts/2015-07-30-r-index.md (hunk 1 above)
Comment (timelyportfolio) [inline _posts/2015-07-30-r-index.md]: I like nit-picky. should be fixed.  Also moved `y` over to line up with `x`

**Comment by cldougl** [comment]
great!

**Pull request merged**